src/NuGet.Core/NuGet.Packaging/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -177,6 +177,10 @@ NuGet.Packaging.Core.PackageDependencyComparer.Equals(NuGet.Packaging.Core.Packa
 NuGet.Packaging.Core.PackageDependencyComparer.GetHashCode(NuGet.Packaging.Core.PackageDependency obj) -> int
 NuGet.Packaging.Core.PackageDependencyComparer.PackageDependencyComparer() -> void
 NuGet.Packaging.Core.PackageDependencyComparer.PackageDependencyComparer(NuGet.Versioning.IVersionRangeComparer versionRangeComparer) -> void
+NuGet.Packaging.Core.PackageDependencyGroupComparer
```

Comment:
Hoping we can remove this, by either moving the implementaiton within the class or making the comparer internal.